.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "C# (.NET)",
+    "build": {
+        "dockerfile": "Dockerfile",
+        "args": {
+            "VARIANT": "3.1"
+        }
+    },
+    "settings": {},
+    "extensions": [
+        "editorconfig.editorconfig",
+        "ms-dotnettools.csharp",
+        "formulahendry.dotnet-test-explorer",
+        "ms-vsliveshare.vsliveshare"
+    ],
+    "forwardPorts": [
+        31234,
```

Comment:
How about adding GraphQL and gRPC ports too?

Reply:
I added more ports, `23061` (maybe GraphQL port), `23142` (maybe gRPC port). 👍🏻